File: testdata/jennies/builders/anonymous_struct/PHPBuilder/src/AnonymousStruct/SomeStructBuilder.php
<?php

namespace Grafana\Foundation\AnonymousStruct;

/**
 * @implements \Grafana\Foundation\Cog\Builder<\Grafana\Foundation\AnonymousStruct\SomeStruct>
 */
class SomeStructBuilder implements \Grafana\Foundation\Cog\Builder
{
    protected \Grafana\Foundation\AnonymousStruct\SomeStruct $internal;

    public function __construct()
    {
    	$this->internal = new \Grafana\Foundation\AnonymousStruct\SomeStruct();
    }

    /**
     * Builds the object.
     * @return \Grafana\Foundation\AnonymousStruct\SomeStruct
     */
    public function build()
    {
        return $this->internal;
    }

    public function time(unknown $time): static
    {
        $this->internal->time = $time;
    
        return $this;
    }

}

File: testdata/jennies/builders/anonymous_struct/JavaBuilder/anonymous_struct/SomeStructBuilder.java
package anonymous_struct;


public class SomeStructBuilder implements cog.Builder<SomeStruct> {
    protected final SomeStruct internal;
    
    public SomeStructBuilder() {
        this.internal = new SomeStruct();
    }
    public SomeStructBuilder time(Object time) {
        this.internal.time = time;
        return this;
    }
    public SomeStruct build() {
        return this.internal;
    }
}
